### ISSUE TITLE
Secondary color

### ISSUE THREAD

**Issue by AlbatracZ** [NoobKingF/nation_bennys#1]
also take note: this script uses rgb on paint and neons.
if your esx does not support it, it wont save the colors of paint.

heres what you can do to make the paint save.

edit functions.lua at es_extended/client/functions.lua

Find:

ESX.Game.GetVehicleProperties = function(vehicle)

add: after color2

rgb = table.pack(GetVehicleCustomPrimaryColour(vehicle)),
rgb2 = table.pack(GetVehicleCustomSecondaryColour(vehicle)),

Find:

ESX.Game.SetVehicleProperties = function(vehicle, props)

add : after props.dirtlevel

if props.rgb then SetVehicleCustomPrimaryColour(vehicle, props.rgb[1], props.rgb[2], props.rgb[3]) end
if props.rgb2 then SetVehicleCustomSecondaryColour(vehicle, props.rgb[1], props.rgb[2], props.rgb[3]) end

**im replace this but secondary color not saving**

**Comment by Tak2607** [comment]
Yep, secondary not saving, if anyone have a solution, would be grateful ;)

**Comment by SpectrumRaijin** [comment]
Secondary color works without any problem https://streamable.com/u3umwl
